components/mautic/actions/create-contact/create-contact.mjs

```diff
@@ -0,0 +1,84 @@
+import mautic from "../../mautic.app.mjs";
+import {
+  pick,
+  pickBy,
+} from "lodash-es";
+import { ConfigurationError } from "@pipedream/platform";
+
+export default {
+  key: "mautic-create-contact",
+  name: "Create Contact",
+  description: "Creates a new contact. [See docs](https://developer.mautic.org/#create-contact)",
+  version: "0.2.0",
```

Comment:
```suggestion
  version: "0.0.1",
```

Reply:
`create-contact` was updated from `create-contacts` (version `0.1.2`).

Reply:
But it's literally a new action, correct?

It's not will be confusing, a new action start with version `0.2.0`?

If I'm wrong just set this as resolved.

Reply:
It's not a new action. I renamed it while refactoring.